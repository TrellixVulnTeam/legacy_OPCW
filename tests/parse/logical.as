true and false;
false or false;
not true or false;
false and true or false and true;
not false or not false or not true;
