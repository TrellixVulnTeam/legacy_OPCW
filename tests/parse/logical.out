ModuleDecl <?> _
  Nodes <?> 
    LogicalAndExpr <?>
      BooleanExpr <?> true
      BooleanExpr <?> false
    LogicalOrExpr <?>
      BooleanExpr <?> false
      BooleanExpr <?> false
    LogicalOrExpr <?>
      LogicalNegateExpr <?>
        BooleanExpr <?> true
      BooleanExpr <?> false
    LogicalAndExpr <?>
      LogicalOrExpr <?>
        LogicalAndExpr <?>
          BooleanExpr <?> false
          BooleanExpr <?> true
        BooleanExpr <?> false
      BooleanExpr <?> true
    LogicalOrExpr <?>
      LogicalOrExpr <?>
        LogicalNegateExpr <?>
          BooleanExpr <?> false
        LogicalNegateExpr <?>
          BooleanExpr <?> false
      LogicalNegateExpr <?>
        BooleanExpr <?> true
